styles/variables-silicon.less

```diff
@@ -16,6 +16,7 @@
 @agate-button-badge-position: -3px 0 auto auto;
 @agate-button-small-height: 48px;
 @agate-button-small-font-size: 24px;
+@agate-button-small-line-height: 1.18;
```

Comment:
Please use a unit in all measurements (`em` in this case).

Reply:
I will undo my changes regarding Button component. It was part of another pull request (#253). Thank you!